src/reconcile/rec_write.c

```diff
@@ -1395,8 +1395,10 @@ __wt_rec_split_finish(WT_SESSION_IMPL *session, WT_RECONCILE *r)
      *
      * Pages with skipped or not-yet-globally visible updates aren't really empty; otherwise, the
      * page is truly empty and we will merge it into its parent during the parent's reconciliation.
+     *
+     * If checkpoint has no entry left to write, reconciliation is done.
```

Comment:
Probably better worded as:
If the checkpoint has no more entries left to write, reconciliation is done.